src-docs/src/views/tables/custom/custom.tsx

```diff
@@ -674,6 +682,7 @@ export default class extends Component<{}, State> {
             mobileOptions={{
               header: column.label,
               ...column.mobileOptions,
+              render: mobileOptionsRender,
```

Comment:
See above - this is basically mimicking what [`EuiBasicTable` does under the hood](https://github.com/elastic/eui/blob/976e603bf55a41e6afe665488aa812e57a3bd47c/src/components/basic_table/basic_table.tsx#L1245-L1246), which is also part of the point of this specific demo example.

```suggestion
              render: column.mobileOptions?.render?.(item),
```